app/lib/features/home/pages/dashboard.dart

```diff
@@ -38,15 +34,15 @@ class Dashboard extends ConsumerWidget {
               ? FloatingActionButtonLocation.miniEndDocked
               : FloatingActionButtonLocation.miniEndFloat,
           floatingActionButtonAnimator: FloatingActionButtonAnimator.scaling,
-          appBar: _buildDashboardAppBar(context, client),
+          appBar: _buildDashboardAppBar(context),
```

Comment:
all unnecessary if we don't have `isGuest` support at this point. nicely removed.